lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyXSDURIResolver.java

```diff
@@ -172,30 +173,17 @@ private String generateServerSchemaXsd(LibertyWorkspace libertyWorkspace, Path s
 
         if (!xsdDestFile.exists()) {
             try {
-                LOGGER.info("Generating schema file from: " + schemaGenJarPath.toString());
                 String xsdDestPath = xsdDestFile.getCanonicalPath();
     
                 LOGGER.info("Generating schema file at: " + xsdDestPath);
-    
-                ProcessBuilder pb = new ProcessBuilder("java", "-jar", schemaGenJarPath.toAbsolutePath().toString(), "--schemaVersion=1.1",
-                        "--outputVersion=2", xsdDestPath, "--locale=" + SettingsService.getInstance().getCurrentLocale().toString()); // Add locale param here
-                pb.directory(tempDir);
-                pb.redirectErrorStream(true);
-                pb.redirectOutput(new File(tempDir, "schemagen.log"));
-    
-                Process proc = pb.start();
-                if (!proc.waitFor(30, TimeUnit.SECONDS)) {
-                    proc.destroy();
-                    LOGGER.warning("Exceeded 30 second timeout during schema file generation. Using cached schema.xsd file.");
-                    return null;
-                }
 
-                if (xsdDestFile.exists()) {
-                    // do some post processing to remove the anyAttribute element from parent element if there is no extraProperties sibling
-                    DocumentUtil.removeExtraneousAnyAttributeElements(xsdDestFile);
-                }
-
-                LOGGER.info("Caching schema file with URI: " + xsdDestFile.toURI().toString());
+                SchemaAndFeatureListGeneratorUtil.generateFile(
```

Comment:
The old code used to return null here if the process for generating the file exceeded the 30 second time out. Now I think you will return to this code with no indication of failure and continue on to the code at 194 that logs a message and returns a String URI for a file that does not exist.